src/entt/entity/handle.hpp

```diff
@@ -71,19 +138,27 @@ struct basic_handle {
     }
 
     /**
-     * @brief Checks if a handle refers to a valid entity or not.
-     * @return True if the handle refers to a valid entity, false otherwise.
-     */
+    * @brief Checks if a handle refers to non-null registry pointer and entity.
+    * @return True if the handle refers to non-null registry and entity, false otherwise.
+    */
     [[nodiscard]] explicit operator bool() const {
+        return (*this != null);
```

Comment:
Here the registry matters instead. What if the entity is a valid one but the registry points to null? Should we consider the handle valid? I don't think so.

Reply:
Just to make sure: this will return `true` only when both registry and entity aren't null. Is this fine or not?

Reply:
I'd say yes. After all, is a handle without a valid registry or a valid entity still valid? You can't use it, so probably not.